Code/src/nlp/native_version.py

```diff
@@ -0,0 +1,33 @@
+import codecs
+import pandas as pd
+import pytrie
+import pickle
+
+
+def load_data_native(path_to_data):
```

Comment:
what exactly do you mean by native?

Reply:
This is a version of the function that uses the data structure(trie) implemented in python, as opposed to the current version, which is written in C++. I was thinking I could change it to suit our purpose better, but this version takes 10 times more memory, so I just left it in for now.